libifupdown/dict.c

```diff
@@ -65,6 +66,31 @@ lif_dict_find(struct lif_dict *dict, const char *key)
 	return NULL;
 }
 
+struct lif_list *
+lif_dict_find_all(struct lif_dict *dict, const char *key)
```

Comment:
We need to add an `lif_dict_free_search_results()` or similar to handle freeing these lists.